components/style/properties/gecko.mako.rs

```diff
@@ -1579,7 +1579,7 @@ fn static_assert() {
         }
     </%self:simple_image_array_property>
 
-    % if shorthand != "background":
+    % if shorthand != "background" and shorthand != "mask":
```

Comment:
nit: `if not shorthand in ["background", "mask"]` may be easier to read and extend.